test/services/api/handlers/projects.js

```diff
@@ -1135,29 +1135,6 @@ experiment('Project Handlers', function() {
       });
     });
 
-    test('update thumbnail (320) succeeds', function(done) {
```

Comment:
What if you left these tests in, but just changed what the thumbnail is equal to? Or are these tests totally redundant now. Maybe just one test?